### DISCUSSION TITLE
taiko-bar-right@2x & taiko-slider@2x were vertically scaled in lazer

### DISCUSSION THREAD

**Discussion by Citrusis** [ppy/osu#30524] [Q&A]
### Type

Game behaviour

### Bug description

Taiko skin works incorrectly in lazer.
The taiko-bar-right@2x & taiko-slider@2x were supposed to be tiled to the bottom of the screen.
But obviously they were vertically scaled.
Wondering whether it is a bug?

### Screenshots or videos

Which works in stable ↓
![stable](https://github.com/user-attachments/assets/6f6c4950-25c8-49cc-98ae-00d7204f4cb6)
Which works in lazer ↓
![lazer](https://github.com/user-attachments/assets/4934e21b-af45-4c3e-b014-d2827da04ecc)

### Version

2024.1009.1

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/17635281/compressed-logs.zip)

**Comment by bdach** [comment]
Please link or attach the skin.

**Comment by Citrusis** [comment]
Here it is. 
[- #Citrusis • Gradients DEMO.zip](https://github.com/user-attachments/files/17641105/-.Citrusis.Gradients.DEMO.zip)

**Comment by peppy** [comment]
The skin is doing weird stuff that we can't support. You can fix it by fixing the skin. Remove the transparent paddings and fix the elements being stretched weirdly.

**Comment by cdwcgt** [comment]
is this dup with https://github.com/ppy/osu/issues/24216?
